src/engraving/accessibility/accessibleroot.cpp

```diff
@@ -119,7 +119,7 @@ void AccessibleRoot::updateStaffInfo(const AccessibleItem* newAccessibleItem, co
 
     if (newAccessibleItem && newAccessibleItem->element()->hasStaff()) {
```

Comment:
It does concern me that `newAccessibleItem->element()` is never nullptr (from what I observed) but  `oldAccessibleItem->element()` can be - nor sure in what cases AssessibleItem::element() should be returning null...